src/apps/AppCenter/DiscoverApps/DiscoverApps.js

```diff
@@ -111,7 +113,7 @@ const TagWrapper = styled.div`
   margin-bottom: 10px;
 `
 
-const Tag = styled(Badge)`
+const TagComponent = styled(Tag)`
   overflow: hidden;
   text-overflow: ellipsis;
   display: block;
```

Comment:
Should this be `flex` now that `Tag` is `inline-flex` by default?

Reply:
Not sure, I updated the places where `Badge` was used to avoid things from breaking.